x-pack/filebeat/module/imperva/securesphere/config/liblogparser.js

```diff
@@ -0,0 +1,2327 @@
+    "c_domain": {to:[{field: "source.domain", setter: fld_prio, prio: 1}]},
+    "c_logon_id": {to:[{field: "user.id", setter: fld_prio, prio: 2}]},
+    "c_user_name": {to:[{field: "user.name", setter: fld_append}]},
+    "c_username": {to:[{field: "user.name", setter: fld_append}]},
+    "cctld": {to:[{field: "url.top_level_domain", setter: fld_prio, prio: 1}]},
+    "child_pid": {convert: to_long, to:[{field: "process.pid", setter: fld_prio, prio: 1}]},
+    "child_pid_val": {to:[{field: "process.title", setter: fld_set}]},
+    "child_process": {to:[{field: "process.name", setter: fld_prio, prio: 1}]},
+    "city.dst": {to:[{field: "destination.geo.city_name", setter: fld_set}]},
+    "city.src": {to:[{field: "source.geo.city_name", setter: fld_set}]},
+    "daddr": {convert: to_ip, to:[{field: "destination.ip", setter: fld_append},{field: "related.ip", setter: fld_append}]},
+    "daddr_v6": {convert: to_ip, to:[{field: "destination.ip", setter: fld_append},{field: "related.ip", setter: fld_append}]},
+    "ddomain": {to:[{field: "destination.domain", setter: fld_prio, prio: 0}]},
+    "devicehostip": {convert: to_ip, to:[{field: "host.ip", setter: fld_prio, prio: 2},{field: "related.ip", setter: fld_append}]},
+    "devicehostmac": {convert: to_mac, to:[{field: "host.mac", setter: fld_prio, prio: 0}]},
+    "dhost": {to:[{field: "destination.address", setter: fld_set}]},
+    "dinterface": {to:[{field: "observer.egress.interface.name", setter: fld_set}]},
+    "direction": {to:[{field: "network.direction", setter: fld_set}]},
+    "directory": {to:[{field: "file.directory", setter: fld_set}]},
+    "dmacaddr": {convert: to_mac, to:[{field: "destination.mac", setter: fld_set}]},
+    "dns.responsetype": {to:[{field: "dns.answers.type", setter: fld_set}]},
+    "dns.resptext": {to:[{field: "dns.answers.name", setter: fld_set}]},
+    "dns_querytype": {to:[{field: "dns.question.type", setter: fld_set}]},
+    "domain": {to:[{field: "server.domain", setter: fld_prio, prio: 0}]},
+    "domain.dst": {to:[{field: "destination.domain", setter: fld_prio, prio: 1}]},
+    "domain.src": {to:[{field: "source.domain", setter: fld_prio, prio: 2}]},
+    "domain_id": {to:[{field: "user.domain", setter: fld_set}]},
+    "domainname": {to:[{field: "server.domain", setter: fld_prio, prio: 1}]},
+    "dport": {convert: to_long, to:[{field: "destination.port", setter: fld_prio, prio: 0}]},
+    "dtransaddr": {convert: to_ip, to:[{field: "destination.nat.ip", setter: fld_prio, prio: 0},{field: "related.ip", setter: fld_append}]},
+    "dtransport": {convert: to_long, to:[{field: "destination.nat.port", setter: fld_prio, prio: 0}]},
+    "ec_outcome": {to:[{field: "event.outcome", setter: fld_set}]},
+    "event_description": {to:[{field: "message", setter: fld_prio, prio: 0}]},
+    "event_time": {convert: to_date, to:[{field: "@timestamp", setter: fld_set}]},
+    "event_type": {to:[{field: "event.category", setter: fld_set}]},
```

Comment:
See comment from other modules, think we shouldn't set this with vendor-specific values. Maybe a better use in the `event.code` or `event.action` fields

Reply:
done